osquery/tables/applications/darwin/browser_plugins.cpp

```diff
@@ -83,32 +89,40 @@ QueryData genBrowserPlugins(QueryContext& context) {
   QueryData results;
   std::vector<std::string> bundles;
 
-  // The caller is not requesting a JOIN against users.
-  // This is "special" logic for user data-based tables since there is a concept
-  // of system-available browser extensions.
-  if (context.constraints["uid"].notExistsOrMatches("0")) {
+  // Lambda to walk through each browser plugin and process the plist file.
+  auto enum_browser_plugins = [&results](fs::path path, std::string uid) {
```

Comment:
Capture a `const &` to these inputs:
```cpp
auto enum_browser_plugins = [&results](const fs::path& path, const std::string& uid) {
```

That'll prevent copying the data into the lambda scope = faster. Passing a `const` makes the function's actions clearer.